SingInTests/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SingInTests
+{
+    internal class SignInTest
+    {
+        private CreateUserRequest request = new CreateUserRequest
+        {
+            FirstName = "Test",
+            LastName = "Test",
+            Birthday = DateTime.Today.AddYears(-20),
+            Email = "testUser@mail.ru",
+            Password = "password"
+        };
+
+        private static void Main(string[] args)
+        {
+            var test = new SignInTest();
+            try
+            {
+                test.TryToSingUp();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("User was created");
+            }
+
+            if (test.TryToSingIn())
+            {
+                Console.WriteLine("Successful");
+            }
+            else
+            {
+                Console.WriteLine("Something was wrong");
+            }
+        }
+
+        private async Task TryToSingUp()
+        {
+
+            const string url = "https://localhost:5011/users/create";
+
+            var client = new RestClient<CreateUserRequest, bool>(url, RestRequestType.POST);
+
+            await client.Execute(request);
```

Comment:
Mb I don`t understand something, but is it normal to create new user in our system in each test? When we run this test, we will add test user to database, Correct me If I`m wrong

Reply:
Every time you run a test, it creates the same user.

Reply:
Okey, probably it`s not bad for sign in tests